src/pages/overview.tsx

```
@@ -0,0 +1,23 @@
+        <div>
+            <h1>Dashboard Overview</h1>
+            <p>You are now logged in. This is the main content area.</p>
+            <button
+                onClick={() => void navigate('/login')}
+                style={{
+                    padding: '10px 20px',
+                    fontSize: '18px',
+                    backgroundColor: 'red',
+                    color: 'white',
+                }}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)